ShipMonitor/ShipMonitor.cs

```diff
@@ -1493,9 +1498,12 @@ private void RemoveModule(Ship ship, string slot, Module replacement = null)
                 }
                 catch (Exception ex)
                 {
-                    Logging.Error("Failed to remove module from ship." + JsonConvert.SerializeObject(new object[] { slot, replacement }), ex);
-
-                    throw;
```

Comment:
Note to self: check that it's OK to do away with this rethrow

Reply:
I think doing away with the rethrow is OK but the rest of the code is flawed: it is modifying `ship` not `updatedShip` and then overwriting `ship` with and unchanged `updatedShip`. Will address in a separate issue.